server/src/main/java/org/apache/iotdb/db/engine/compaction/utils/log/CompactionLogger.java

```diff
@@ -30,47 +29,41 @@
 import java.util.List;
 
 /** MergeLogger records the progress of a merge in file "merge.log" as text lines. */
-public class RewriteCrossSpaceCompactionLogger implements AutoCloseable {
+public class CompactionLogger implements AutoCloseable {
 
-  public static final String COMPACTION_LOG_NAME = "cross-compaction.log";
-  public static final String COMPACTION_LOG_NAME_FEOM_OLD = "merge.log";
+  public static final String CROSS_COMPACTION_LOG_NAME = "cross-compaction.log";
+  public static final String CROSS_COMPACTION_LOG_NAME_FEOM_OLD = "merge.log";
+  public static final String INNER_COMPACTION_LOG_NAME = ".compaction.log";
```

Comment:
From my understanding(I am not very familiar with Merge now...), the value of `INNER_COMPACTION_LOG_NAME ` should be something like "inner-compaction.log" if we change the value of `CROSS_COMPACTION_LOG_NAME ` to "cross-compaction.log" ?

Reply:
Resolved.